effect/README.md

```diff
@@ -131,10 +131,10 @@ val result = openFile("data.json").attempt.map {
 }
 ```
 
-You can submerge failures with `IO.absolve`, which turns an `IO[Throwable \/ A]` into an `IO[A]`:
+You can submerge failures with `IO.absolve`, which turns an `IO[Void, E \/ A]` into an `IO[E, A]`:
 
 ```scala
-def sqrt(io: IO[Double]): IO[Double] =
+def sqrt(io: IO[E, Double]): IO[E, Double] =
   IO.absolve(
```

Comment:
If absolve only accepts `IO[Void, E \/ A]`, how does this code typecheck?

Reply:
I do like the `Done` one.